### PR TITLE
feat: improved cockpit textures and screen reflections

### PR DESCRIPTION
**Pull request by tyler58546**
## Summary of Changes

Improved cockpit textures and screen reflections by @MoreRightRudder 
- Changed all interior textures with significantly darkened AO
- Changed the glass material and dust textures
- All panels were masked using real photos and PBR associated with the panels matches the photo in use

## Screenshots (if necessary)

image
image
image

## Testing instructions

> WOOOOOO!
Pls test for at least 15 minutes to see if the sim crashes thx
Let me know where you see errors within the textures or areas for improvement

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by MoreRightRudder** [comment]
WOOOOOO!

Pls test for at least 15 minutes to see if the sim crashes thx

Let me know where you see errors within the textures or areas for improvement

File: src/model/glass/glass.gltf [hunk 1]
```diff
@@ -767,7 +767,8 @@
     "extensionsUsed": [
         "ASOBO_normal_map_convention",
         "MSFT_texture_dds",
-        "ASOBO_asset_optimized"
+        "ASOBO_asset_optimized",
+        "ASOBO_material_kitty_glass"
```
Comment (tracernz) [inline src/model/glass/glass.gltf:771]: meow

Comment (lolisec) [reply src/model/glass/glass.gltf:771]: i like kittens

Comment (tyler58546) [reply src/model/glass/glass.gltf:771]: 🐱

**Comment by wpine215** [comment]
Reflections seems to be almost nonexistent/too subtle, compared to initial previews.

image

Dust on the PFD seems to be a little bit too opaque/exaggerated in my opinion, at least on the altitude/speed/VS tapes.

image


IRL Reference photos:
image

image

Also, not sure if these textures are able to have their resolution bumped up to the resolution of the others (unless it's a model limitation). Not a huge issue but just stood out to me:

image


Otherwise, all other parts of the texture look perfect to me.

**Comment by FinalLightNL** [comment]
**Discord:** FinalLight#2113
**Object of testing:** #5497 
**Tier of Testing:** 2
**Date:** (01/08/2021)

**Testing Process:** Loaded up the aircraft, took a look around, did a full flight. (CTD'd taxiing to the gate, but that's an SU5 issue)
**Negatives:** I personally think the reflections are too subtle, the previews were more reflective (maybe a bit too much) i think a middle ground between this and the preview would be nice.
In flight i don't see the reflections at all (see first screenshot at FL380), when parked in some angles i do see them on the ground.
The leather on top of the FCU seems less detailed than it was before.
**Testing Results:** Passed

**Conclusions:**  The textures are incredible, i thought it couldn't get better than it was but bravo Mike!
The green hue on the overhead panel is practically gone now, the dust and greased spots look excellent.
The knobs having some wear to them is a nice touch as well.
Apart from the reflections and the leather on top of the FCU that spans all across i think it's a 10/10!

**Media:**
FL380 flight in bright sunlight-
FL380

FCU-
FCU

FCU Leather-
FCU Leather

Pedestal-
pedestal

Captain side-
captain seat

Panels-
panels

Overhead-
overhead

Overhead close up-
overhead close

MCDU-
mcdu

PFD showing the dust clearly, love it!-
PFD

Knobs with wear-
knobs

Cockpit with lights in test mode-
test all lights on

Overhead with test lights on-
test all lights on 2

**Comment by MoreRightRudder** [comment]
> Non Quality Assurance Tester Report
> 
> **Discord:** FinalLight#2113
> **Object of testing:** #5497
> **Tier of Testing:** 2
> **Date:** (01/08/2021)

Sounds good I am re-redoing the leather and glass now

**Comment by FlyingCarpet7** [comment]
Great work with the new textures. Have not tried the PR, just looked at the screenshots above.
Some of the knobs are very seldom touched by the crew and does not generate much wear at all. I can see several places where I really would like to reduce the wear on the knobs. Let me know if you would like me to specify which knobs are affected more and less by wear. 
And now I just realised that we probably only have one texture setup for a specific type of knob, and therefore it will be replicated and look the same on several places? Anyways, some of them are unique and could have a bit less wear.

**Comment by MoreRightRudder** [comment]
> Great work with the new textures. Have not tried the PR, just looked at the screenshots above.
> Some of the knobs are very seldom touched by the crew and does not generate much wear at all. I can see several places where I really would like to reduce the wear on the knobs. Let me know if you would like me to specify which knobs are affected more and less by wear.
> And now I just realised that we probably only have one texture setup for a specific type of knob, and therefore it will be replicated and look the same on several places? Anyways, some of them are unique and could have a bit less wear.

It is the same repeating texture yep. I can reduce it though

**Comment by MoreRightRudder** [comment]
> Great work with the new textures. Have not tried the PR, just looked at the screenshots above.
> Some of the knobs are very seldom touched by the crew and does not generate much wear at all. I can see several places where I really would like to reduce the wear on the knobs. Let me know if you would like me to specify which knobs are affected more and less by wear.
> And now I just realised that we probably only have one texture setup for a specific type of knob, and therefore it will be replicated and look the same on several places? Anyways, some of them are unique and could have a bit less wear.

FlightSimulator_2021-08-02_00-54-06

**Comment by MoreRightRudder** [comment]
> Reflections seems to be almost nonexistent/too subtle, compared to initial previews.
> Dust on the PFD seems to be a little bit too opaque/exaggerated in my opinion, at least on the altitude/speed/VS tapes. (might need to zoom in to this photo/open in another tab to see)
> IRL Reference photos:
> 
> Also, not sure if these textures are able to have their resolution bumped up to the resolution of the others (unless it's a model limitation). Not a huge issue but just stood out to me:
> 
> Otherwise, all other parts of the texture look perfect to me.

**Increased reflections, decreased dust over parts of the pfds, redid leather on FCU, reduced knobs wear, increased white color on knobs, new plastic on side window, added 8K albedo, comp, and normal textures for the Glareshield, and SideTop (window) textures.**  
FlightSimulator_2021-08-02_01-25-21
FlightSimulator_2021-08-02_01-26-00
FlightSimulator_2021-08-02_01-27-03

**Comment by FlyingCarpet7** [comment]
I think I mentioned before that the efis range knob has a poor surface quality and quite quickly receives wear, already after a year or so in service. 

BCB36E73-13AD-498A-B378-67E8B3478CBA

**Comment by MoreRightRudder** [comment]
> I think I mentioned before that the efis range knob has a poor surface quality and quite quickly receives wear, already after a year or so in service.

Unfortunately this area is repeated on all similar knobs including and above the autopilot unit. So this is not an option. I am changing the color of the knobs again though.

**Comment by MoreRightRudder** [comment]
I updated the knobs color (darkened it) changed VOR selector switches material.

**Comment by wpine215** [comment]
Everything looks perfect to me - all issues have been fixed, no crashes either.

**Review by wpine215 — APPROVED** [review]

**Pull request merged**